config/checkstyle_input_suppressions.xml

```diff
@@ -1382,6 +1382,8 @@
             files="javadoc[\\/]summaryjavadoc[\\/]InputSummaryJavadocPeriodAtEnd.java"/>
   <suppress id="ConfigCommentOnTopInputs"
             files="javadoc[\\/]summaryjavadoc[\\/]InputSummaryJavadocPeriod.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]summaryjavadoc[\\/]InputSummaryJavadocHtmlFormat.java"/>
```

Comment:
you should not have changes in file `checkstyle_input_suppressions.xml` it is suppressions for old files only.
All new files should follow requirements for Input files.

Reply:
Done